zerver/lib/import_realm.py

```diff
@@ -1995,3 +2008,54 @@ def add_users_to_system_user_groups(
         )
         for membership in usergroup_memberships
     )
+
+
+def check_migration_status(migration_status_file: str) -> None:
+    # Each server might have a different set of apps installed (e.g., Zulip Cloud has
+    # zilencer/corporate, but self-hosted might not). This function only checks for
+    # migration mismatches in apps installed on both servers.
```

Comment:
I only see an `if local_migrations.get(app)` check, not one for `exported_migrations` being empty.